src/test/java/dog/pawbook/logic/commands/EditDogCommandTest.java

```diff
@@ -50,7 +54,7 @@ public void execute_allFieldsSpecified_success() {
         EditDogCommand.EditDogDescriptor descriptor = new EditDogDescriptorBuilder(editedDog).build();
```

Comment:
actually this one can statically import

Reply:
Also while you're at it, perhaps change the 2 lines above to directly retrieve ID_TWO which is the same dog, then can get rid of all the poorly named variables.

Reply:
Done.